### DISCUSSION TITLE
SA-1 Reading issue on HW V4

### DISCUSSION THREAD

**Discussion by TCorgard82** [sanni/cartreader#490] [Need help]
Need some advise on this. I have the Clock Gen enabled in the firmware per the guide, I get the right freq with the Clock Calibration. I did install a SuperCIC programmed with the Hex file from the guide, but I just get errors when trying to read any of my SA-1 carts. Other SNES games read just fine.

Has anyone seen this issue, or know a solution?

**Comment by kamalawala** [comment]
Obligatory "did you clean the cartridge thoroughly with isopropyl alcohol and q-tips". Other than that, could it be related to this? #481

**Reply by TCorgard82** [reply]
yes my carts are clean, it ended up being I have a Rev 1 6 slot board with incorrect traces, so i have to modify my board.

**Comment by TCorgard82** [comment]
So I modified my Rev 1 board and it finally reads my SA-1 game  Mario RPG, but when pulling the rom I get Chksum...Error 9232 and CRC32... 08CF10DE -> Not found. So the rom runs fine in ZSNES, but does say the CRC and checksum is wrong, I used IpsAndSum application, repaired the checksum and all is good.

**Comment by sanni** [comment]
Do you have the battery installed in your HW4? If not try adding a 470uF capacitor between GND and VCC near the Snes slot.

**Comment by TCorgard82** [comment]
So I tried again to day with a 470uF, and it would not read the cart, I put
back my .1uF cap, and read the cart again and pulled the rom and no errors.
And correct crc and checksum. So my guess was a fluke. Also means maybe a
470uF is too much for the CIC.

**Comment by TCorgard82** [comment]
I keep all my carts clean, I use 1Up Cards and 99% isopropyl. I will check
on my 6 slot adapter which rev I purchased, and maybe that is my issue.
Thank you.

**Comment by TCorgard82** [comment]
Battery installed and USB charger installed, and I used a 0.1uF like the
CIC board said, but I will change to a 470uF and see what happens.
